src/components/node-list/styles/_panels.scss

```diff
@@ -2,6 +2,23 @@
 @import '../../../styles/mixins';
 @import '../../../styles/variables';
 
+.pipeline-nodelist__filter-panel {
+  z-index: 1;
```

Comment:
Don't we have a z-index scale as scss variables?

Reply:
Not currently but probably a good idea for a future refactor?